src/content/docs/apm/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure.mdx

```diff
@@ -4,12 +4,59 @@ tags:
+
+You can install the Node.js agent onto your Azure-managed web apps with the New Relic Azure Site Extension for Node. This doc walks you through compatibility, installation, and configuration to get your Azure web app data into New Relic.
+
+## Compatibility and requirements [#compatibility-requirements]
+
+Azure Site Extensions are only available for Windows-based App Service apps that are deployed as code. For applications running on Windows 32, full Code Level Metrics support (file path, line, column) is not available. Profiling falls back to function name only.
+
+## Install the Node.js agent with Azure Site Extension
+
+To install the New Relic Azure Site Extension, add the `NewRelic.Azure.WebSites.Extension.NodeAgent` site extension from the Azure home page. Follow these steps:
+
+1. From the Azure Home page, click the App Services tile, then select the target application in the displayed list.
+2. Scroll down to <DNT>**Extensions**</DNT> in the options listed on the left under the <DNT>**Development Tools**</DNT> category.
+3. Click on <DNT>**+ Add**</DNT> at the top of the page to toggle the extension drop down, then select <DNT>**New Relic Node Agent**</DNT>. Check the box for accepting the legal terms
+4. Click <DNT>**Add**</DNT> on the bottom of the page to start installing the extension.
+
+Once installed, the extension creates the following artifacts:
+
+- Folder: `C:\home\SiteExtensions\NewRelic.Azure.Websites.Extension.NodeAgent`
+- XDT: `applicationHost.xdt` that will add the necessary `NODE_OPTIONS` environment variable on application startup
+- The New Relic Node agent and dependencies will be installed into `C:\home\site\wwwroot\node_modules`
+
+If the New Relic agent has been installed successfully and logging has been enabled, the agent will append its logs to a file at `C:\home\site\wwwroot\newrelic_agent.log`. If the extension fails to install, a log file is created at `C:\home\SiteExtensions\NewRelic.Azure.Websites.Extension.NodeAgent\install.log`.
+
+## Configure the Node.js agent [#configure-agent]
+
+The New Relic Node agent is configured with the `newrelic.js` file, or via environment variables. [See our documentation for more detailed configuration](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/).
+
+Once the site extension is installed, you'll need to manually enter one configuration item before restarting your application.
+  - On the options listed on the left, scroll down to "Environment variables" located under the "Settings" category and add the following:
+    - `NEW_RELIC_LICENSE_KEY` - Your New Relic license key value
+
+The Node agent automatically adds the `NODE_OPTIONS` environment variable with a value of `-r newrelic` which starts the agent.
+  - Note: Any previously defined `NODE_OPTIONS` will be removed and reset with `-r newrelic`.
+
+  <Callout variant="important">
```

Comment:
Per our style guide, we want to limit the number of iMPORTANT tips. At best it can cause "Tip" exhaustion and cause readers to glaze past them, at worst it might indicate a product-side problem (that is, if there are a lot of important tips that can devastate install, it gives the impression that the product is fragile). Can we choose one important tip and change the rest to regular sentences?

Reply:
I'd added the site extension text (with its own Important blocks) to this page, which definitely makes this worse. Instead, I'm separating it out and cross linking. With that change, this Azure hosting page would be restored to its previous number of important blocks. (Which may still be too many, but that's a separate issue.)